travis-ci/vmtest/run.sh

```diff
@@ -436,11 +436,17 @@ sudo umount "$mnt"
 
 echo "Starting VM with $(nproc) CPUs..."
 
-qemu-system-x86_64 -nodefaults -display none -serial mon:stdio \
-	-cpu kvm64 -enable-kvm -smp "$(nproc)" -m 4G \
-	-drive file="$IMG",format=raw,index=1,media=disk,if=virtio,cache=none \
-	-kernel "$vmlinuz" -append "root=/dev/vda rw console=ttyS0,115200$APPEND"
-
+if kvm-ok ; then
+  qemu-system-x86_64 -nodefaults -display none -serial mon:stdio \
+    -cpu kvm64 -enable-kvm -smp "$(nproc)" -m 4G \
+    -drive file="$IMG",format=raw,index=1,media=disk,if=virtio,cache=none \
+    -kernel "$vmlinuz" -append "root=/dev/vda rw console=ttyS0,115200$APPEND"
+else
+  qemu-system-x86_64 -nodefaults -display none -serial mon:stdio \
+	  -cpu qemu64 -machine accel=tcg -smp "$(nproc)" -m 4G \
+  	-drive file="$IMG",format=raw,index=1,media=disk,if=virtio,cache=none \
+	  -kernel "$vmlinuz" -append "root=/dev/vda rw console=ttyS0,115200$APPEND"
```

Comment:
Oh, that's a lot of copy/paste of a very long list of arguments... Why not extract what differs into a variable. Something along these lines:

```
if kvm-ok then
QEMU_CPU_ARGS=-cpu kvm64 -enable-kvm
else
QEMU_CPU_ARGS=-cpu qemu64 -machine accel=tcg
fi
qemu-system-x86_64 .... $(QEMU_CPU_ARGS) ....
```
?